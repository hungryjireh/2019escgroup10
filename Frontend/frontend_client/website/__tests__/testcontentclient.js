import React from "react";
import { render, cleanup, fireEvent, waitForElement, getByLabelText, getByText, getByTestId } from 'react-testing-library';
import ContentClient from "../components/Hero/Bot/ContentClient";
import {ClientFace, ClientText} from "../components/Hero/Bot/ContentClient";

afterEach(cleanup);

it("ClientFace renders", () => {
    const { asFragment } = render(<ClientFace />);
    expect(asFragment()).toMatchSnapshot();
});

it("ClientText renders", () => {
    const { asFragment } = render(<ClientText />);
    expect(asFragment()).toMatchSnapshot();
});