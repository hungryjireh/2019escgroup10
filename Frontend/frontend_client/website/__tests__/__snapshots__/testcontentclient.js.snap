// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClientFace renders 1`] = `
<DocumentFragment>
  <img
    class="sc-bwzfXH kQZLzn"
  />
</DocumentFragment>
`;

exports[`ClientText renders 1`] = `
<DocumentFragment>
  <p
    class="sc-htpNat fvOSVV"
  />
</DocumentFragment>
`;
